lms/static/scripts/frontend_apps/components/FilePickerApp.tsx

```diff
@@ -250,6 +254,11 @@ export default function FilePickerApp({ onSubmit }: FilePickerAppProps) {
   const [title, setTitle] = useState(
     promptForTitle ? 'Hypothesis assignment' : null,
   );
+
+  const [assignmentGradableMaxPoints, setIsAssignmentGradableMaxPoints] =
```

Comment:
The setter should be `setAssignmentGradableMaxPoints`.

```suggestion
  const [assignmentGradableMaxPoints, setAssignmentGradableMaxPoints] =
```